mailit/tests/email_parser/incoming_mail_tests.py

```diff
@@ -368,6 +368,36 @@ def test_it_marks_the_outbound_message_as_an_error(self):
         outbound_message = OutboundMessage.objects.get(id=self.outbound_message.id)
         self.assertEquals(outbound_message.status, "error")
 
+class FlaggingBouncedMail(TestCase):
+    def setUp(self):
+        super(FlaggingBouncedMail, self).setUp()
+        self.message = Message.objects.get(id=1)
+        self.contact = Contact.objects.get(value="mailnoexistente@ciudadanointeligente.org")
+        self.outbound_message = OutboundMessage.objects.create(message=self.message, contact=self.contact, site=Site.objects.get_current())
+        identifier = OutboundMessageIdentifier.objects.get(outbound_message=self.outbound_message)
+        identifier.key = "4aaaabbb"
```

Comment:
Where is this `4aaaabbb` key from? It is sensitive or secret?

Reply:
This is also pombola upstream default tests - I would leave as is for now.